docs/topics/native/native-cocoapods-dsl-reference.md

```diff
@@ -0,0 +1,143 @@
+[//]: # (title: Cocoapods Gradle plugin DSL reference)
+
+Kotlin Cocoapods Gradle plugin is a tool for creating podspec files. The generated podspec integrates your project with
+[CocoaPods dependency manager](https://cocoapods.org/).
+
+Here we provide a reference of its contents; use it as a reminder when working with the CocoaPods integration.
+
+* Learn how to [set up the environment](native-cocoapods.md#set-up-the-environment-to-work-with-cocoapods) and
+[configure Kotlin Cocoapods Gradle plugin](native-cocoapods.md#add-and-configure-kotlin-cocoapods-gradle-plugin).
+* Depending on your project and purposes, you can add dependencies between [a Kotlin project and a Pod library](native-cocoapods-libraries.md)
+as well as [a Kotlin Gradle project and an Xcode project](native-cocoapods-xcode.md).
+
+
+## ID and version
+
+To apply the CocoaPods plugin, add the following lines to the `build.gradle(.kts)` file:
+
+```kotlin
+plugins {
+   kotlin("multiplatform") version "%kotlinVersion%"
+   kotlin("native.cocoapods") version "%kotlinVersion%"
+}
+```
+
+The plugin versions match the Kotlin release versions. The most recent version is %kotlinVersion%.
+
+## The `cocoapods` block
+
+The `cocoapods` block is the top-level block for the Cocoapods configuration. Inside it, you can use the following blocks,
+functions and properties:
+
+| **Name**                              | **Description**                                                                                                                                                                                                               | 
+|---------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `version`                             | The version of the pod. If not specified, a Gradle project version is used. If none of these properties are configured, you'll get an error.                                                                                  |
```

Comment:
AFAIR, we omit the dots in the table sentences... double-check, please

Reply:
I couldn't find any guidelines for this. In my experience, when you have full sentences, especially more than one in a cell, you put dots at the end